### ISSUE TITLE
Is there a way to configure the default "shell" directory?

### ISSUE THREAD

**Issue by voxik** [rpm-software-management/mock#1303]
Using `mock shell` quite often, I always endup in `/builddir`, the user home directory. But this is not very useful default (to me). Is there a way to configure different default such as `/builddir/build/BUILD/` ? I was trying e.g. `config_opts['environment']['HOME'] = '/builddir/build/BUILD/', but without success

**Comment by xsuchy** [comment]
No, not possible. And I would rather avoid to have option for this. Because IIRC there is some logic to clean builddir but preserve homedir (to keep certs and files from copyin).

**Comment by voxik** [comment]
It is possible to change homedir via `config_opts['chroothome']`, but that is not what I am asking. I'd like to automate something like:

~~~
$ mock shell

<mock-chroot> sh-5.2$ cd /builddir/build/BUILD/
~~~

Actually there is `--cwd DIR` option, so I'd like to make it persistent
